markdown_lab/utils/chunk_utils.py

```
@@ -215,4 +215,4 @@ def create_semantic_chunks(
         )
         chunks.append(chunk)
 
-    return chunks
+    return chunks
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)